edge/pkg/metamanager/process.go

```diff
@@ -132,7 +132,14 @@ func (m *metaManager) processInsert(message model.Message) {
 	}
 
 	// Notify edged
-	sendToEdged(&message, false)
+	switch resType {
+	case "pod":
+		if podMatchByNodeNameOrHostIP(&message, metaManagerConfig.Config.NodeName, metaManagerConfig.Config.NodeIP) {
+			sendToEdged(&message, false)
+		}
+	default:
+		sendToEdged(&message, false)
+	}
```

Comment:
We can wrap a function to avoid duplication of code

Reply:
yes, wrapped the a func sendToEdgedByNodeNameOrHostIP(message model.Message, resType string, isSync bool)